package.json

```diff
@@ -80,14 +86,18 @@
     "postcss-loader": "^1.2.1",
     "postcss-nested": "^1.0.0",
     "postcss-simple-vars": "^3.0.0",
-    "react": ">=15.0.0",
```

Comment:
Lets' move this back to a more generic react version unless we specifically know we don't support the other versions.

Reply:
Not sure what is meant by "move back." Do you have a suggestion for what version? We added 15.4.2 because of peer dependencies.